backend/src/authorisation.js

```diff
@@ -0,0 +1,27 @@
+import { allow, rule, shield } from 'graphql-shield'
```

Comment:
Didn't know there was already an allow from graphql-shield 👍 
Now we also used it in our permissions 😄